lib/gap_intelligence/models/download.rb

```diff
@@ -1,9 +1,11 @@
 module GapIntelligence
   class Download < Record
     attributes :start_at, :end_at, class: Date
+    attribute :created_at, class: Time
```

Comment:
why do we need created_at, I did not see api returning created_at from downloads endpoint